rest/route/service_test.go

```diff
@@ -688,25 +688,9 @@ func TestTestPaginator(t *testing.T) {
 				limit := 100
 				expectedTests := []model.Model{}
 				for i := testToStartAt; i < testToStartAt+limit; i++ {
-					status := "pass"
-					if i%2 == 0 {
-						status = "fail"
-					}
-					nextModelTest := &model.APITest{
-						Id:        utility.ToStringPtr(mgobson.ObjectId(fmt.Sprintf("object_id_%d_", i)).Hex()),
-						StartTime: model.ToTimePtr(time.Unix(0, 0)),
-						EndTime:   model.ToTimePtr(time.Unix(0, 0)),
-						Status:    utility.ToStringPtr(status),
-						TaskId:    utility.ToStringPtr(""),
-						TestFile:  utility.ToStringPtr(""),
-						Logs: model.TestLogs{
-							URL:            utility.ToStringPtr(""),
-							URLRaw:         utility.ToStringPtr(""),
-							LogId:          utility.ToStringPtr(""),
-							RawDisplayURL:  utility.ToStringPtr("/test_log//0?test_name=&group_id=&text=true"),
-							HTMLDisplayURL: utility.ToStringPtr("/test_log//0?test_name=&group_id=#L0"),
-						},
-					}
+					nextModelTest := &model.APITest{}
+					_ = nextModelTest.BuildFromService(&cachedTests[i])
```

Comment:
They populate the `APITest` with the expected output. It would only fail if an unrecognized type is passed in, but since we know both `string` and `*testresult.TestResult` are accepted there is no point of checking the error.